src/es/components/pages/StandortDetailPage.html

```diff
@@ -0,0 +1,356 @@
+    </div>
+    <aside>
+        <div class="aside-image">
+        </div>
+    </aside>
+</o-grid>
+
+<o-grid namespace="grid-2columns-content-section-" first-container-vertical="" first-column-with="66%" with-border="" width="100%">
+    <ks-o-body-section content-width-var="100%" no-margin-y>
+        <div class="center">
+            <ks-a-button namespace="button-primary-" color="secondary" href="#angebote">
+                234 Angebote
+                <a-icon-mdx icon-name="ArrowDownRight" size="1em" class="icon-right"></a-icon-mdx>
+            </ks-a-button>
+        </div>
+        <!-- Expand -->
+        <mdx-component class="margin-y-m">
+            <mdx-expand
+                line-clamp="5"
+                label-closed="Mehr anzeigen"
+                label-open="Weniger anzeigen"
+                expanded="false"
+            >
+                <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. </p>
+            </mdx-expand>
+        </mdx-component>
+        <!-- /Expand -->
+        <!-- CMS Akkordeon Container -->
+        <div class="margin-y-m">
+        <m-details namespace="details-default-icon-right-">
+                <details>
+                    <summary>
+                        <h4>accordion 1</h4>
+                    </summary>
+                    <div>
+                        <style>
+                            :host details summary~* {
+                                --details-default-color-secondary: var(--color-hover);
+                            }
```

Comment:
Okay, but better be solved in Details.js with maybe an extra class, because it appears several times.